portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,85 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin: 20px;
 }
+#nav_bar{
+    display: flex;
+    flex-direction: row;
+}
+#scroll_buttons{
+    margin-left: auto;
+}
+.flex_hor{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+}
+
+button{
+    background-color: white;
+    border: none;
+    padding: 15px;
+    margin: 5px;
+    border-radius: 25px;
+    font-size: medium;
+    font-weight: 400;
+    border-color: antiquewhite;
+    color:#fca61f; 
```

[4.2.4](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space after a property name’s colon.

```suggestion
    color: #fca61f; 
```